### PR TITLE
Feature/api ingredient

### LINKED ISSUE #18: Change how we are accessing data with fetch requests

### LINKED ISSUE #20: CREATE a file called apiCalls.js
* here we place every fetch get and post

### PR DESCRIPTION
**Pull request by rhen92**
# Description of Changes

- Finish api fetch requests

# Linked Issues

- Resolves #18, resolves #20

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const userGET = () => fetch('http://localhost:3001/api/v1/users')
+  .then(response => response.json()).catch(err => console.log('ERROR'));
+
+export const ingredientsGET = () => fetch('http://localhost:3001/api/v1/ingredients')
+  .then(response => response.json()).catch(err => 'Ingredient Data ERROR');
+
+export const recipeGET = () => fetch('http://localhost:3001/api/v1/recipes')
+  .then(response => response.json()).catch(err => console.log('Recipe Error!'));
```
Comment (mistercanderson) [inline src/apiCalls.js:8]: I like how you wrapped & exported the `fetch()` calls within a function expression here

File: src/scripts.js [hunk 2]
```diff
@@ -1,6 +1,20 @@
-import users from './data/users-data';
-import recipeData from  './data/recipe-data';
-import ingredientData from './data/ingredient-data';
+import { userGET, ingredientsGET, recipeGET } from './apiCalls'
```
Comment (mistercanderson) [inline src/scripts.js:1]: Cool separation of API calls into its own file here, very modular 👏

File: src/scripts.js [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+  users = data.map(users => new User(users));
+})
+
+ingredientsGET().then(data => {
+  ingredientData = data;
+})
+
+recipeGET().then(data => {
+  recipeData = data;
+})
```
Comment (mistercanderson) [inline src/scripts.js:17]: We could try and refactor these similar `.then()` chains into a `Promise.all()` in the future

File: src/scripts.js [hunk 4]
```diff
@@ -47,9 +56,15 @@ searchBtn.addEventListener("click", searchRecipes);
 showPantryRecipes.addEventListener("click", findCheckedPantryBoxes);
 searchForm.addEventListener("submit", pressEnterSearch);
 
+function onLoad () {
```
Comment (mistercanderson) [inline src/scripts.js:59]: This is nice for cutting down on eventListeners

**Review by mistercanderson — COMMENTED** [review]
Looks great, left some code comments, nice work!

File: test/recipe-test.js [hunk 5]
```diff
@@ -36,15 +36,7 @@ describe('Recipe', function() {
   });
 
   it('should initialize with an array of ingredients', function() {
```
Comment (battan40) [inline test/recipe-test.js:38]: If theres time later, lets review the tests for data importation

**Review by battan40 — APPROVED** [review]

**Pull request merged**